hack/update-swagger-spec.sh

```diff
@@ -51,16 +51,20 @@ TEMP_DIR=${USE_TEMP:-$(mktemp -d /tmp/openshift-cmd.XXXX)}
 export CURL_CA_BUNDLE="${TEMP_DIR}/openshift.local.config/master/ca.crt"
 
 # Start openshift
+echo "Starting OpenShift..."
 pushd "${TEMP_DIR}" > /dev/null
-OPENSHIFT_ON_PANIC=crash openshift start master --listen="https://0.0.0.0:8443" --master="https://127.0.0.1:8443" 1>&2 &
+OPENSHIFT_ON_PANIC=crash openshift start master --listen="https://0.0.0.0:8443" --master="https://127.0.0.1:8443" >/dev/null 2>&1 &
```

Comment:
> I read that doesn't work on all shells?

Citation?  It's used in `test-end-to-end.sh` so if it's busted it should be fixed there too.